components/XGFEDatePicker.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import DatePicker from 'react-native-datepicker'
+
+export default class XGFEDatePicker extends Component {
```

Comment:
What is XGFE?

Reply:
just plugin name, is u don't like, lll change renderDatePicker.